### ISSUE TITLE
Does not work if Moodle runs in a subdirectory of the web server root

### ISSUE THREAD

**Issue by mudrd8mz** [moodleuulm/moodle-local_maintenance_livecheck#1]
Due to this line https://github.com/moodleuulm/moodle-local_maintenance_livecheck/blob/master/amd/src/livecheck.js#L29 the called ajax.php is not found if the Moodle itself is installed in a subdirectory of the domain, e.g. https://my.university.de/elearning/

**Comment by mudrd8mz** [comment]
This patch fixes the issue:
```
--- livecheck.js.orig   2017-05-19 16:53:58.839976018 +0200
+++ livecheck.js        2017-05-19 16:59:24.462964854 +0200
@@ -6,7 +6,7 @@
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
-define(['jquery', 'core/str', 'core/log'], function($, str, log) {
+define(['jquery', 'core/str', 'core/log', 'core/config'], function($, str, log, config) {
     "use strict";
 
     // Global variable for left time.
@@ -26,7 +26,7 @@
         // We know about the benefits of the core/ajax module (https://docs.moodle.org/dev/AJAX),
         // but for this very lightweight check we only use a simple jQuery AJAX call.
         $.ajax({
-            url: '/local/maintenance_livecheck/ajax.php',
+            url: config.wwwroot + '/local/maintenance_livecheck/ajax.php',
             dataType: 'json',
             type: 'POST',
             data: {
```

**Comment by abias** [comment]
Oh, shame on me, I missed that. Thanks for spotting, David - this is fixed now in eec377385eee7ba37d2091de7c6723726ef7ed42.
